packages/expo-sensors/src/Barometer.ts

```diff
@@ -0,0 +1,11 @@
+import DeviceSensor from './DeviceSensor';
+import ExponentBarometer from './ExponentBarometer';
+
+type PressureMeasurement = {
```

Comment:
```suggestion
type BarometerMeasurement = {
```

There are other types of pressure, like pressure on the touch screen and the pressure of unread GitHub notifications